examples/nextjs-blog/pages/index.js

```diff
@@ -14,8 +15,51 @@ import { getImageAlt, getDateString } from '../utils/post/post.util'
 import { DEFAULT_POST_IMAGE_URL } from '../constants/constants'
 import { containerWide } from '../components/styled'
 
-const Home = function(props) {
-  function renderPostCards(posts) {
+class Home extends PureComponent {
+  constructor(props) {
+    super(props)
+    this.state = {
+      posts: null,
+      home: null,
+    }
+    this._postsSubscription = null
+    this._homeSubscription = null
+  }
+
+  componentDidMount() {
+    this._postsSubscription = Home.getPostData(true, (error, response) => {
```

Comment:
any reason why you don't use `this.getPostData`?

and also why do you declare methods on Class after creating it and not on the class? I know it does the same, just curious as to your preference.

Reply:
I guess because its's a "utility", it can be shared between instances. I didn't give it too much though. Seems appropriate.